Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift

```diff
@@ -292,10 +292,16 @@ public final class PackageInfoMapper: PackageInfoMapping {
             }
         )
 
+        let sanitizedPackageName = if packageInfo.name.contains(" ") {
+            "\"\(packageInfo.name)\""
+        } else {
+            packageInfo.name
+        }
```

Comment:
Can we add a test for this in the `PackageInfoMapperTests`?

Reply:
I've added a new test to `PackageInfoMapperTests`, it's hopefully what you expect )))